index.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <title>Tribute page</title>
+</head>
+<body>
+    <div id ="main">
+        <div id="title">
+            <h1>Dr. Norman Borlaug</h1>
+            <p>The man who saved a billion lives</p>
+        </div>
+        <div id="img-div">
+            <img id="image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" alt="Dr. Norman Borlaug seen standing in Mexican wheat field with a group of biologists">
+            <caption id="image-caption">
+                Dr. Norman Borlaug, third from the left, trains biologists in Mexico
+                on how to increase wheat yields - part of his life-long war on hunger.
+              </caption>
+        </div>
```

Comment:
I think If you use sections instead of divs to enhance readability (Semantic tags):-
- main div include{section, section, section...}